homeassistant/components/onkyo/quality_scale.yaml

```
@@ -0,0 +1,89 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: exempt
+    comment: |
+      This integration uses a push API. No polling required.
+  brands: done
+  common-modules: done
+  config-flow:
+    status: todo
+    comment: |
+      The data_descriptions are missing.
+  config-flow-test-coverage:
+    status: todo
+    comment: |
+      Coverage is 100%, but the tests need to be improved.
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration do not explicitly subscribe to events.
+  entity-unique-id: done
+  has-entity-name: todo
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable: todo
+  integration-owner: done
+  log-when-unavailable: todo
+  parallel-updates: todo
+  reauthentication-flow:
+    status: exempt
+    comment: |
+      This integration does not require authentication.
+  test-coverage: todo
+  # Gold
+  devices: todo
+  diagnostics: todo
+  discovery: todo
+  discovery-update-info: todo
+  docs-data-update: todo
+  docs-examples: todo
+  docs-known-limitations: done
+  docs-supported-devices: done
```

```suggestion
  docs-supported-devices: todo
```

While there is a mention in the introduction of the integration, a list of support devices is missing.

---

We support all the network receivers from the dates mentioned in the introduction. Having an explicit list of all the receiver models produced in that time frame, would only, in my opinion:
- take a lot of space and therefore obscure more important information
- actually confuse users and make them question whether their receiver is supported, if it is not on the list (and sourcing such a list in a complete fashion would obviously be very challenging)

Therefore I think the current state of the documentation is actually preferable. It correctly conveys our intention: as long as you buy a new enough network receiver, you should expect it to be supported in Home Assistant.